### DISCUSSION TITLE
Firmware file for Swann NVR8-7400 & NHD-818/819

### DISCUSSION THREAD

**Discussion by vmallet** [AT0myks/reolink-fw-archive#17] [Firmware sharing]
This is the firmware file we talked about on vmallet/pakler#8. 

I believe this might have come from this page: 
https://support.swann.com/hc/en-us/articles/4639228186649-NVR8-7400-NHD-818-819-Firmware-Release

[NVR8-7400_1705_3438_1103.pak.zip](https://github.com/AT0myks/reolink-fw-archive/files/11527270/NVR8-7400_1705_3438_1103.pak.zip)

**Comment by AT0myks** [comment]
Ah, it's a Swann firmware, that explains why I couldn't find it. For now I'm not gonna add it but maybe in the future the repo could be expanded to include that brand too. Thank you for getting back to me.
